src/utils/configValidators/allValidate.js

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+export default (...validators) => {
+  if (validators.length < 2) {
+    throw Error("allValidate requires at least 2 validators");
+  }
+
+  return (...args) => {
+    let error;
+    let i;
+    for (i = 0; i < validators.length; i += 1) {
+      error = validators[i](...args);
```

Comment:
Consider doing `const error = ...` here instead of declaring `error` away from its usage. You could do the same thing with `for (let i`. I used to do what you've done here with `var`, but stopped with `let`/`const`. I suspect you may have a different perspective on these things though, which is fine.

Reply:
This is a for loop not a forEach, so I can't do `const error = ...`.  I'm using a for loop because I want to short circuit the return.  I will put the let inside the for.

Reply:
You can use const in a `for` loop since the `for` iteration creates a new scope for `const`.